### ISSUE TITLE
s3Copy doesn't exist?

### ISSUE THREAD

**Issue by akrikos** [jenkinsci/pipeline-aws-plugin#98]
Hi folks, I'm trying to run a multipart pipeline deploy that uploads to one bucket at one point in the process (which works fine) and then copy from that bucket to another one later in the process.

Using v1.26 of this plugin, I'm getting the following error (snipped out steps that don't start with 's3'):
```
java.lang.NoSuchMethodError: No such DSL method 's3Copy' found among steps [
...
s3Delete, s3Download, s3FindFiles, s3PresignURL, s3Upload
...
]
```

Is this something wrong with the package or my installation? How can I tell?

I tried to look at differences between s3Copy & s3Upload but didn't see anything significant (at least to my untrained eye with this codebase).

While I can work around this by re-uploading or any number of other things, it is annoying to not be able to copy. Any help would be appreciated.

**Comment by hoegertn** [comment]
should be released now with 1.27

**Comment by akrikos** [comment]
OH! Apologies. I'd assumed it had already been released but just looked at the commits and found where it was added. I'll be more careful next time.
